### DISCUSSION TITLE
Could not poll the database: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: No address associated with hostname

### DISCUSSION THREAD

**Discussion by Alex-Goaga** [firefly-iii/firefly-iii#4565] [Firefly III]
Just tried to get firefly-iii working on remote server (proxmox) with docker-compose on debian 10 container.

The error : 
Could not poll the database: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: No address associated with hostname (SQL: select `id`, `name`, `data` from `configuration` where `name` = is_demo_site and `configuration`.`deleted_at` is null limit 1)

To keep in mind, my linux knowleage would be considerated 2/10 and i follow each line as its displayed.

Following the instructions from https://docs.firefly-iii.org/firefly-iii/installation/docker/  (the Using Docker Compose chapter) :

- installed docker (for debian 10 from https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-debian-10) and docker-compose (using tutorial here : https://docs.docker.com/compose/install/ )
- installed portainer for a gui interface to see them (not actually using it) from here https://documentation.portainer.io/v2.0/deploy/ceinstalldocker/
- saved the docker-compose file to a folder as following the tutorial from https://www.youtube.com/watch?v=8vsLx1losCE ( , edited with notepad the mysql password)
- saved the env file to the same folder and renamed it .env , edited with notepad the mysql password
- ran docker-compose -f docker-compose.yml up -d from inside that folder

After a few minutes navigated to lan ip of the machine.
I have no idea what to do more from here.

Added entire log downloaded from portainer

[_firefly3_fireflyiii_1_logs.txt](https://github.com/firefly-iii/firefly-iii/files/6228612/_firefly3_fireflyiii_1_logs.txt)
[_firefly3_fireflyiiidb_1_logs.txt](https://github.com/firefly-iii/firefly-iii/files/6228613/_firefly3_fireflyiiidb_1_logs.txt)

**Comment by JC5** [comment]
Docker had failed to start the containers. The DB container seems to have no storage space.

**Reply by Alex-Goaga** [reply]
Thank you for the fast reply. I had a few minutes at work and i tried again with a fresh new machine only this time , added more space (20 GB instead of 12 GB) , using the same tutorial and still had the same issue . I've redone manually the db container now its starting but the same errors are found in the new log 
[_firefly3_fireflyiiidb_1_logs (1).txt](https://github.com/firefly-iii/firefly-iii/files/6229993/_firefly3_fireflyiiidb_1_logs.1.txt)

**Reply by JC5** [reply]
Nah it has to be something else. Even 12GB should be enough. Replace the following:

```
    image: yobasystems/alpine-mariadb:latest
```

With:
```
    image: mariadb
```

And see what happens.

**Reply by Alex-Goaga** [reply]
It WOOOOORRKKKSSS like a charm, everything installs as described and the dbs are working. I made a few tests and i havent found any problems
